modules/features/settings/src/main/java/au/com/shiftyjelly/pocketcasts/settings/whatsnew/WhatsNewFragment.kt

```diff
@@ -1,107 +1,260 @@
-import au.com.shiftyjelly.pocketcasts.views.helper.ToolbarColors
-import au.com.shiftyjelly.pocketcasts.views.helper.WhatsNew
-import au.com.shiftyjelly.pocketcasts.views.helper.WhatsNewPage
 import dagger.hilt.android.AndroidEntryPoint
+import au.com.shiftyjelly.pocketcasts.images.R as IR
 import au.com.shiftyjelly.pocketcasts.localization.R as LR
 
 @AndroidEntryPoint
 class WhatsNewFragment : BaseFragment() {
 
-    private var binding: FragmentWhatsnewBinding? = null
-
-    private val pageChangeCallback = object : ViewPager2.OnPageChangeCallback() {
-        override fun onPageSelected(position: Int) {
-            val binding = binding ?: return
-            binding.pageIndicatorView.position = position
-            binding.btnNext.text = getString(if (position < WhatsNew.pages.count() - 1) LR.string.settings_whats_new_next else LR.string.settings_whats_new_done)
+    override fun onCreateView(
+        inflater: LayoutInflater,
+        container: ViewGroup?,
+        savedInstanceState: Bundle?,
+    ): View =
+        ComposeView(requireContext()).apply {
+            setBackgroundColor(Color.Transparent.toArgb())
+            setContent {
+                AppTheme(theme.activeTheme) {
+                    setViewCompositionStrategy(ViewCompositionStrategy.DisposeOnViewTreeLifecycleDestroyed)
+                    val onClose: () -> Unit = {
+                        @Suppress("DEPRECATION")
+                        activity?.onBackPressed()
+                    }
+                    WhatsNewComposable(
+                        onGoToSettings = {
+                            onClose()
+                            goToPlaybackSettings()
+                        },
+                        onClose = onClose,
+                    )
+                }
+            }
         }
-    }
 
-    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View? {
-        binding = FragmentWhatsnewBinding.inflate(inflater, container, false)
-        return binding?.root
+    private fun goToPlaybackSettings() {
+        val fragmentHostListener = activity as? FragmentHostListener
+            ?: throw IllegalStateException("Activity must implement FragmentHostListener")
+        val fragment = PlaybackSettingsFragment.newInstance(scrollToAutoPlay = true)
+        fragmentHostListener.addFragment(fragment)
     }
 
-    override fun onDestroyView() {
-        super.onDestroyView()
-        binding?.mainViewPager?.unregisterOnPageChangeCallback(pageChangeCallback)
-        binding = null
+    companion object {
+        fun isWhatsNewNewerThan(versionCode: Int?): Boolean {
+            return Settings.WHATS_NEW_VERSION_CODE > (versionCode ?: 0)
+        }
     }
+}
 
-    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
-        super.onViewCreated(view, savedInstanceState)
+@OptIn(ExperimentalAnimationApi::class)
+@Composable
+private fun WhatsNewComposable(
```

Comment:
Is it weird to use Composable in the component name? Would Page be better. This is a super important comment, feel free to ignore it. 😄

Reply:
> Is it weird to use Composable in the component name? Would Page be better. 

I did mean to update that name and just forgot. Thanks for catching it. Updated to Page in [e84b9c7](https://github.com/Automattic/pocket-casts-android/pull/1170/commits/e84b9c7a7f87ec7fe7ba0ac7a03c7904cc215df0).